tools/gdal/gdal_macros.xml

```diff
@@ -0,0 +1,112 @@
+##	            }
+##           </citation>
+##           <citation type="bibtex">
+##            @inproceedings{kerbiriou2010long,
+##            title={Long term trends monitoring of bats, from biodiversity indicator production to species specialization assessment},
+##            author={Kerbiriou, C and Bas, Y and Dufr{\^e}ne, L and Robert, A and Julien, JF},
+##            booktitle={24th Annual Meeting of the Society for Conservation Biology, Edmonton, Canada},
+##            year={2010}
+##            }
+##            </citation>
+##            <citation type="doi">10.5334/jors.154</citation>  
+##        </citations>
+##    </xml>
+
+
+    <xml name="gdal_advanced_params_select">
+            <param name="advanced" type="select" label="Specify advanced parameters">
+                <option value="simple" selected="true">No, use program defaults.</option>
+                <option value="advanced">Yes, see full parameter list.</option>
+            </param>        
+            <when value="simple">
+            </when>
+    </xml>
+
+    <xml name="gdal_param_separate">
+                <param name="separate" type="boolean" truevalue="-separate" falsevalue="" checked="false" label="-separate" />
+    </xml>
+
+
+    <xml name="gdal_param_b">
+                <param name="b" type="text" label="-b band" value="e.g : -b 1 -b 2 -b 3"/>
+    </xml>
+
+
+    <xml name="gdal_param_oo">
+                <param name="oo" type="text" label="-oo" value="NAME=VALUE" />
+    </xml>
+
+
+    <xml name="gdal_param_r">
+                <conditional name="condi_resample">
+                    <param label="-r" name="resample" type="boolean" truevalue="resampling" falsevalue="no_resampling" checked="false"/>
+                        <when value="resampling">
+                            <param name="r" type="select" label="Reseampling algorithm" >
+                                <option value="nearest" selected="true"/>
+                                <option value="bilinear"/>
+                                <option value="cubic"/>
+                                <option value="cubicspline"/>
+                                <option value="lanczos"/>
+                                <option value="average"/>
+                                <option value="mode"/>
+                                <option value="max"/>
+                                <option value="min"/>
+                                <option value="med">Median resampling</option>
+                                <option value="q1">First quartile resampling</option>
+                                <option value="q2">Third quartile resampling</option>
+                            </param>
+			</when>
+                        <when value="no_resampling">
+                        </when>
+                </conditional>
+    </xml>
+
+
+    <xml name="gdal_param_te ">
+                    <param name="te" type="select" label="-te">
```

Comment:
indentation here and everywhere :)

Reply:
I'm wonderning for indentation in macros.. does it need to match current alignment in the macro file, or alignment in the wrapper where it will be called ?

Reply:
In the macro file.